docs/sources/k6/next/javascript-api/k6-browser/console-message.md

```diff
@@ -11,7 +10,7 @@ weight: 03
 
 ## Supported APIs
 
-| Method                                                                                                          | Playwright Relevant Distinctions |
+| Method                                                                                                          | Relevant Distinctions |
```

Comment:
@ankur22 for this mention of Playwright, do we need to create new pages for each row on this table so we can remove this?